front/components/assistant/conversation/input_bar/InputBarContainer.tsx

```diff
@@ -31,6 +31,7 @@ export interface InputBarContainerProps {
   owner: WorkspaceType;
   selectedAssistant: AgentMention | null;
   stickyMentions: AgentMention[] | undefined;
+  hideQuickActions: boolean;
```

Comment:
Let add some default props:

```
InputBarContainer.defaultProps = {
hideQuickActions: false
};
```

Reply:
Not needed IMHO, there's only one call site, that actually uses the prop; I prefer to add code when it becomes needed if you don't mind (+ even if we reused it's debatable whether the prop should be implicitely set everywhere or not)

Reply:
Fine.